modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -479,4 +518,115 @@ export default class NestedTabs extends Base {
 
 		return !! $tabTitleContainerElement && isTabTitleActive ? true : false;
 	}
+
+	changeScrollStatus( event ) {
```

Comment:
I am thinking possibly we could add a comment to the codepen that we used for this, for reference for future maintenance.

Reply:
Done already?